testsuite/features/upload_files/01-netcfg.yaml

```diff
@@ -0,0 +1,13 @@
+network:
+  version: 2
+  renderer: networkd
+  ethernets:
+    ens3:
+      match: {}
+      dhcp4: yes
+    ens4:
+      match: {}
+      dhcp4: yes
+      nameservers:
+      search: [DOMAINS]
+      addresses: [ADDRESSES]
```

Comment:
Missing end-of-line on last line. Your development environment has this problem.

Reply:
Sure. Fixed. Thanks to let me know, after change to my new laptop had again to configure IntelliJ and Sublime.